modules/auxiliary/scanner/http/wordpress_scanner.rb

```diff
@@ -126,7 +128,37 @@ def run_host(target_host)
         end
         print_status("#{target_host} - Finished scanning plugins")
       end
-      print_status("#{target_host} - Finished all scans")
+      #      if 1==1
+      if datastore['USERS']
+        print_status("#{target_host} - Searching Users")
+        url = wordpress_url_rest_api + '/users'
+        res = send_request_cgi({
+          'method' => 'GET',
+          'uri' => url
+        })
+        if res.nil?
+          print_error('Error getting response.')
+        elsif res.code == 200
+          parsed = JSON.parse(res.body)
```

Comment:
There's a more convenient method to accomplish this using the [`#get_json_document`](https://github.com/rapid7/metasploit-framework/blob/d52f039fa683a21ad570c3a69392f7ce20d0dcfd/lib/rex/proto/http/response.rb#L123). It has the added benefit of performing error handling and defaulting to an empty hash when it can't be parsed.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16061/commits/2d9edcd22f45cfe495188cbb0007eeee4b69cf25

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16061/commits/2d9edcd22f45cfe495188cbb0007eeee4b69cf25